emerald-lang-spec/2num-operators.em

```diff
@@ -0,0 +1,32 @@
+ (+ 5 4)
+ ; compiles to:: 5 + 4
```

Comment:
Love these files. I wonder if you could actually make tests out of these in future.